R/check_heterogeneity.R

```diff
@@ -84,7 +84,7 @@ print.check_heterogeneity <- function(x, ...) {
 
 
 #' @keywords internal
-.n_unique <- function(x, na.rm = TRUE) {
+insight::n_unique <- function(x, na.rm = TRUE) {
```

Comment:
![haha](https://user-images.githubusercontent.com/26301769/158436283-6fd70748-de4e-439a-980d-a64d763c22b1.gif)

Reply:
Huh, there were two copies of this function in the package: one in `utils.R` and another in `check_heterogeneity.R`